/**:
  ros__parameters:
    speed_to_erpm_gain: -4614.0
    speed_to_erpm_offset: 0.0
    steering_angle_to_servo_gain: -1.2135
    steering_angle_to_servo_offset: 0.5304
    port: /dev/ttyACM0
    duty_cycle_min: 0.0
    duty_cycle_max: 0.0
    current_min: 0.0
    current_max: 100.0
    brake_min: -20000.0
    brake_max: 200000.0
    speed_min: -23250.0
    speed_max: 23250.0
    position_min: 0.0
    position_max: 0.0
    servo_min: 0.15
    servo_max: 0.85

vesc_to_odom_node:
  ros__parameters:
    odom_frame: odom
    base_frame: base_link
    publish_tf: true
    use_servo_cmd_to_calc_angular_velocity: false
    wheelbase: .25

throttle_interpolator:
  ros__parameters:
    rpm_input_topic: commands/motor/unsmoothed_speed
    rpm_output_topic: commands/motor/speed
    servo_input_topic: commands/servo/unsmoothed_position
    servo_output_topic: commands/servo/position
    max_acceleration: 2.5
    throttle_smoother_rate: 75.0
    max_servo_speed: 3.2
    servo_smoother_rate: 75.0
